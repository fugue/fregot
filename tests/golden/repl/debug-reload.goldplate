{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${GOLDPLATE_NAME}.stderr"},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ],
    "stdin": [
        "rule {true}",
        ":break rule",
        "rule",
        ":reload"
    ]
}
