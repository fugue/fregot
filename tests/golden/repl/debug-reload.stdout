repl% repl% repl% 1| rule {true}
         [32m^^^^[0m
repl(debug)% repl(debug)% 